linux/utils/get_linux_system_info.yml

```diff
@@ -16,6 +16,14 @@
     - guest_os_release.ID is defined
     - guest_os_release.ID == 'rhcos'
 
+- name: "Correct OS distribution info for Miracle Linux 8.x"
+  ansible.builtin.set_fact:
+    guest_os_ansible_distribution_ver: "{{ guest_os_release.VERSION | regex_search('\\d\\.\\d') }}"
+    guest_os_ansible_distribution_minor_ver:  "{{ guest_os_release.VERSION.split()[0].split('.')[-1] }}"
+  when: 
+    - guest_os_ansible_distribution == "MIRACLE"
+    - guest_os_ansible_distribution_major_ver | int  == 8
+
```

Comment:
1. Please change the task name from `Correct OS distribution info` to `Correct OS distribution version`
2. Use `guest_os_release.VERSION | regex_search('\\d\\.\\d')` as the 3rd when condition to execute this task
3. Set `guest_os_ansible_distribution_ver` by using `split` like `guest_os_ansible_distribution_minor_ver`